spec/lib/event_publisher_spec.rb

```diff
@@ -29,78 +39,89 @@ def mock_connection(params)
     allow(@channel).to receive(:fanout).and_return(@exchange)
     allow(@exchange).to receive(:name).and_return('exchange name')
 
+    allow(@queue).to receive(:bind)
+    # allow(@queue).to receive(:name).and_return('queue name')
```

Comment:
Could this be removed? Or commented to say what it's waiting on/why it isn't working